docs/2.8.0/docs/app-dev/bindings-java/open-tracing.rst

```diff
@@ -51,31 +51,12 @@ Add Project Dependencies
 
 To use the OpenTelemetry libraries, add the following **Maven** dependencies to your project's ``pom.xml``:
 
-::
-
-    <dependency>
-      <groupId>io.opentelemetry</groupId>
-      <artifactId>opentelemetry-api</artifactId>
-      <version>x.y.z</version>
-    </dependency>
-    <dependency>
-      <groupId>io.opentelemetry</groupId>
-      <artifactId>opentelemetry-sdk</artifactId>
-      <version>x.y.z</version>
-    </dependency>
-    <dependency>
-      <groupId>io.opentelemetry</groupId>
-      <artifactId>opentelemetry-exporter-jaeger</artifactId>
-      <version>x.y.z</version>
-    </dependency>
-    <dependency>
-      <groupId>io.opentelemetry.instrumentation</groupId>
-      <artifactId>opentelemetry-grpc-1.6</artifactId>
-      <version>x.y.z</version>
-    </dependency>
+.. remoteliteralinclude:: https://raw.githubusercontent.com/digital-asset/ex-java-bindings-with-opentelemetry/master/pom.xml
+   :language: xml
+   :lines: 33-52
 
 .. note::
-    Replace ``x.y.z`` for both dependencies with the version that you want to use. You can find the available versions by checking the `Maven Central Repository <https://search.maven.org/artifact/io.opentelemetry/opentelemetry-api>`__ etc.
+    Replace the version number in all dependencies with the version that you want to use. You can find the available versions by checking the `Maven Central Repository <https://search.maven.org/artifact/io.opentelemetry/opentelemetry-api>`__ etc.
```

Comment:
```suggestion
    Replace the version number in each dependency with the version you want to use. To find available versions, check the `Maven Central Repository <https://search.maven.org/artifact/io.opentelemetry/opentelemetry-api>`__.
```

Reply:
The Maven search doesn't show anything for me... maybe double-check the link

Reply:
It gets me to this page:
![image](https://github.com/digital-asset/docs.daml.com/assets/28315767/2f9af8a4-d181-4565-b56c-317a51780d42)

Reply:
I have also asked a few colleagues to check, and it works for them too, :thinking:

Reply:
It also works for me when I click it from the current documentation page (little note at the bottom):
https://docs.daml.com/2.8.0/app-dev/bindings-java/open-tracing.html#add-project-dependencies

Reply:
Thanks, the link works fine for me now. I must have hit a Maven hiccup or outage yesterday.